app/Console/Commands/AddFirstLibrariand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use Illuminate\Console\Command;
+
+class AddFirstLibrariand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'librarian:add';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Add the first librarian to the system';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $name = $this->ask('Enter librarian name');
+        $username = $this->ask('Enter librarian username');
+        $email = $this->ask('Enter librarian email');
+        $jmbg = $this->ask('Enter librarian jmbg');
+        $password = $this->ask('Enter librarian password min 8 and max 16 characters');
+
+        // ... ask for last name, email, username, jmbg and validate them
+
+        $librarian = new User;
+        $librarian->name = $name;
+        $librarian->username = $username;
+        $librarian->email = $email;
+        $librarian->password = $password;
```

Comment:
I've never seen this approach before, can you jsut verify that it works? I'll test it later when I get the chance as well, but I've never seen "hashed" cast being used to hash passwords before setting them.

User.php
`    protected function casts(): array
    {
        return [
            'email_verified_at' => 'datetime',
            'password' => 'hashed',
        ];
    }`

Reply:
This wokrs. I checked.